app/src/main/java/com/vocabulary/myvocabulary/repositories/share/ShareDictionaryRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.vocabulary.myvocabulary.repositories.share
+
+import android.net.Uri
+import io.reactivex.Observable
+import io.reactivex.Single
+
+interface ShareDictionaryRepository {
+    fun storeCsvData(data: Uri)
+    fun getCsvData(): Uri?
```

Comment:
This is not getting the Data. It is getting an URI.

Reply:
Fixed.